rakelib/plugin.rake

```diff
@@ -61,6 +66,18 @@ namespace "plugin" do
     task.reenable # Allow this task to be run again
   end
 
+  task "remove-non-oss-plugins" do |task, _|
+    puts("[plugin:remove-non-oss-plugins] Removing non-OSS plugins")
+
+    LogStash::RakeLib::OSS_EXCLUDED_PLUGINS.each do |plugin|
+      remove_plugin(plugin)
```

Comment:
Just a note that for now this approach works but if in the future any non-OSS plugin has a dependency, we'll have one or two issues to be aware of:
1. non-OSS may introduce dependencies that will not be used by Logstash
2. if the non-OSS brings a non-OSS dependency we'll have to remove it as well.